frontend/src/routes/manage/Video/VideoDetails.tsx

```diff
@@ -98,3 +105,12 @@ const VideoButtonSection: React.FC<{ event: AuthorizedEvent }> = ({ event }) =>
     </ButtonSection>;
 };
 
+const VideoMetadataSection: React.FC<{ event: AuthorizedEvent }> = ({ event }) => {
+    const [commit, inFlight] = useMutation<VideoDetailsMetadataMutation>(updateEventMetadata);
+
+    return <MetadataSection
+        item={{ ...event, description: event.description }}
+        {...{ commit, inFlight }}
+    />;
```

Comment:
Maybe in the definition of `MetadataSection` one could make `commit` non-optional and remove all the code paths for the "not able to modify". But dunno, maybe we need that in the future again? :thinking:  I mean, we will have to change that component anyway to add more metadata fields at some point. So eh, dunno, maybe don't change anything now :D

Reply:
Hm it was originally only optional because metadata updates weren't possible for videos before this PR. So I think making this non-optional wouldn't be a bad idea and easy enough to revert anyway.